packages/flutter/lib/src/rendering/sliver_grid.dart

```diff
@@ -0,0 +1,426 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+import 'box.dart';
+import 'object.dart';
+import 'sliver.dart';
+import 'sliver_multi_box_adaptor.dart';
+
+/// Describes the placement of a child in a [RenderSliverGrid].
+///
+/// See also:
+///
+///  * [SliverGridDelegate.getGeometryForChildIndex], which returns this object
+///    to describe the child's placement.
+///  * [RenderSliverGrid], which uses this class during its
+///    [RenderSliverGrid.performLayout] method.
+class SliverGridGeometry {
+  /// Creates an object that describes the placement of a child in a [RenderSliverGrid].
+  const SliverGridGeometry({
+    this.scrollOffset,
+    this.crossAxisOffset,
+    this.mainAxisExtent,
+    this.crossAxisExtent,
+  });
+
+  /// The scroll offset of the leading edge of the child.
```

Comment:
Is this relative to the leading edge of the grid sliver (the edge at which scroll offset is zero)?